src/CMakeLists.txt

```diff
@@ -590,6 +592,11 @@ if(BUILD_BITCOIN_ZMQ)
 	target_link_libraries(server zmq)
 endif()
 
+if(BUILD_BITCOIN_PLUGIN_NNG)
+	add_subdirectory(plugin_nng)
+	target_link_libraries(server plugin_nng)
```

Comment:
https://cmake.org/cmake/help/latest/command/else.html

Can add a stub in here if it's off so you don't need conditionals in the rest of the code.